src/exceptions.cc

```diff
@@ -128,7 +128,9 @@ Local<Value> UVException(Isolate* isolate,
         String::Concat(isolate, js_msg, FIXED_ONE_BYTE_STRING(isolate, "'"));
   }
 
-  Local<Object> e = Exception::Error(js_msg)->ToObject(isolate);
+  Local<Object> e =
+    Exception::Error(js_msg)->ToObject(isolate->GetCurrentContext())
```

Comment:
`env->context()` ?

Reply:
This is part of the public API, and it’s possible for embedders/addons to enter another context before calling this → `GetCurrentContext()` is correct.